cas-server-core/src/main/java/org/jasig/cas/services/DefaultRegisteredServiceAccessStrategy.java

```diff
@@ -60,6 +62,12 @@
     /** Is the service allowed to use SSO? **/
     private boolean ssoEnabled = true;
 
+    private String startingDateTime;
```

Comment:
With the current design, you can only plug in one access strategy into a service.  Are you saying there could be `TimeBasedRegisteredServiceAccessStrategy` that extends from `DefaultRegisteredServiceAccessStrategy`?